cmd/librarian/main.go

```diff
@@ -24,12 +24,6 @@ import (
 
 func main() {
 	ctx := context.Background()
-
-	log.Println("Arguments passed to librarian:")
```

Comment:
This has been really useful when diagnosing CI stuff - but I understand it's not useful for a lot of interactive work. Would adding a -v option or similar be feasible?

Reply:
I added a log line later on if that works?